spot_rl_experiments/experiments/skill_test/test_sem_place.py

```diff
@@ -14,38 +21,59 @@
 
     if in_fre_lab:
         # at FRE
-        place_target = "coffee_table"
+        place_target = "160_teddy bear"
     else:
         # at NYC
         place_target = "test_desk"
 
     spotskillmanager = SpotSkillManager(use_mobile_pick=False, use_semantic_place=True)
-
     is_local = False
-    if enable_estimation_before_place:
-        place_target = None
-        is_local = True
-
     # Start testing
     contnue = True
+    INITIAL_ARM_JOINT_ANGLES = [0, -180, 180, 90, 0, -90]
+    episode_ctr = 0
+    # Get EE Pose Initial in rest position
+    spot_pos, spot_ort = spotskillmanager.spot.get_ee_pos_in_body_frame()
+    # Set Orientation as Zero
+    spot_ort = np.zeros(3)
     while contnue:
+        # Open Gripper
+        spotskillmanager.spot.open_gripper()
+        input("Place an object in Spot's gripper and press Enter to continue...")
+        # Place Object and Close Gripper
         rospy.set_param("is_gripper_blocked", 0)
-        spotskillmanager.place(place_target, is_local=is_local, visualize=True)
-        contnue = map_user_input_to_boolean("Do you want to do it again ? Y/N ")
+        episode_log = {"actions": []}  # mypy: ignore-errors
+        spotskillmanager.spot.close_gripper()
+        input("waiting for user to get ready with camera")
+        if enable_estimation_before_place:
+            is_local = True
+            spotskillmanager.place(
+                place_target,
+                is_local=is_local,
+                visualize=False,
+                enable_waypoint_estimation=True,
+            )
+        else:
+            spotskillmanager.place(
+                place_target,
+                is_local=is_local,
+                visualize=True,
```

Comment:
why is visualize=True for this one? 

we can also change the whole if/else block to:
```
if enable_estimation_before_place:
    place_target = None
spotskillmanager.place(
    place_target,
    is_local=enable_estimation_before_place,
    visualize=False,
     enable_waypoint_estimation=enable_estimation_before_place,
)
```